site/en/blog/prerender-pages/index.md

```diff
@@ -0,0 +1,263 @@
+---
+# Required
+layout: 'layouts/blog-post.njk'
+title: Prerender pages in Chrome for instant page navigations
+description: |
+ ??
+authors:
+ - tunetheweb
+date: 2022-11-28
+#updated: 2022-11-28
+hero: image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/eohdiqaZlxnWen7TT66M.jpg
+alt: City road at dusk with a long exposure of car lights giving impression of speed
+tags:
+ - performance
+ - chrome-108
+---
+
+As [announced previously](https://web.dev/speculative-prerendering/), the Chrome team has been working on options to bring back full prerendering of future pages that a user is likely to navigate to. This modern reboot of prerendering", and from there on simply prerendering will start rolling out from Chrome 108.
+
+## A brief history of prerender
+
+In the past, Chrome supported the `<link rel="prerender" href="/next-page/">` resource hint, however it was not broadly supported beyond Chrome, and it wasn’t a very expressive API.
+
+This legacy prerendering via the link `rel=prerender` hint was deprecated in favor of [no-state prefetch](/blog/nostate-prefetch/), which instead fetched the resources needed by the page, but did not fully prerender the page nor execute JavaScript. No-state prefetch does help improve page performance by improving the resource loading, but will not deliver an _instant_ page load like a full prerender would.
+
+The Chrome team is now ready to reintroduce full prerendering back into Chrome. To avoid complications with existing usage, and to allow for future expansion of prerendering, this new prerender mechanism will not use the `<link rel="prerender"...>` syntax, which remains in place for no-state-prefetch, with a view of retiring this at some point in the future.
```

Comment:
```suggestion
The Chrome team is now ready to reintroduce full prerendering back into Chrome. To avoid complications with existing usage, and to allow for future expansion of prerendering, this new prerender mechanism will not use the `<link rel="prerender"...>` syntax, which remains in place for NoState Prefetch, with a view of retiring this at some point in the future.
```